packages/ui/input/checkbox/index.css

```diff
@@ -69,3 +69,7 @@
 .switch .input {
   border-radius: calc(2.5 * var(--gridSize));
 }
+
+.switch .input:focus {
+  box-shadow: inset 0 0 0 var(--color-switch-slider_checked_hover), 0 0 0 2px var(--color-switch-slider_checked_hover);
```

Comment:
@AlexBabusanu why not remove this box shadow entirely? rely on an 2px outline with `--color-border-active` for the container node instead ?

Reply:
i will do that, but there is a difference between `--color-border-active` and `--color-switch-slider_checked_hover` so i will keep the one i use

Reply:
Oh, ok, I'll check, I didn't notice it.